tests/regression/37-congruence/03-interval-overflow.c

```diff
@@ -0,0 +1,44 @@
+// PARAM: --enable ana.int.interval --enable ana.int.congruence --disable ana.int.def_exc
+// Overflow information should be passed from the interval domain to congruences
+#include <assert.h>
+#include <stdio.h>
+
+int main(){
+  char r;
```

Comment:
Maybe switch to `signed char` in this example, it is not really clear from the C standard if char should be signed or not (it is signed for GCC).